e2e_test/shared.sh

```diff
@@ -1,6 +1,8 @@
 #!/bin/bash
 #shellcheck disable=SC2034  # unused vars make sense in a shared file
 
+SCRIPT_DIR=$(readlink -f "$(dirname "$0")")
+export SCRIPT_DIR
```

Comment:
For conciseness and consistency with the lines below, we could do 
```suggestion
export SCRIPT_DIR=$(readlink -f "$(dirname "$0")")
```

Reply:
shellcheck advised me to do it like that because it otherwise would [mask the return value of the cmd](https://www.shellcheck.net/wiki/SC2155)

Reply:
Ah that one. Makes sense.